frontend/app/services/personal-information-service.server.ts

```diff
@@ -151,41 +246,44 @@ function createPersonalInformationService() {
     const homeAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Home');
     const mailingAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Mailing');
 
     return {
-      applictantId: personalInformationApiResponse.Client.ClientIdentification ? personalInformationApiResponse.Client.ClientIdentification.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID : '',
-      clientId: personalInformationApiResponse.Client.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client Number').at(0)?.IdentificationID,
-      firstName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonGivenName?.at(0),
-      lastName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonSurName?.at(0),
-      emailAddress: personalInformationApiResponse.Client.PersonContactInformation?.EmailAddress.EmailAddressID,
-      phoneNumber: personalInformationApiResponse.Client.PersonContactInformation?.TelephoneNumber.FullTelephoneNumber,
+      applicantCategoryCode: personalInformationApiResponse.BenefitApplication.Applicant?.ApplicantCategoryCode.ReferenceDataID,
+      applictantId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID,
+      clientId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client ID').at(0)?.IdentificationID,
+      clientNumber: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText == 'Client Number').at(0)?.IdentificationID,
+      birthDate: personalInformationApiResponse.BenefitApplication.Applicant?.PersonBirthDate.dateTime,
+      firstName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonGivenName?.at(0),
+      lastName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonSurName,
+      emailAddress: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.EmailAddress.EmailAddressID,
+      maritalStatus: personalInformationApiResponse.BenefitApplication.Applicant?.PersonMaritalStatus.StatusCode.ReferenceDataID,
       homeAddress: homeAddressList
         ?.map((aHomeAddress) => ({
           streetName: aHomeAddress.AddressStreet.StreetName,
           cityName: aHomeAddress.AddressCityName,
-          provinceName: aHomeAddress.AddressProvince.ProvinceName,
-          countryName: aHomeAddress.AddressCountry.CountryName,
-          countryCode: aHomeAddress.AddressCountry.CountryCode,
+          provinceCode: aHomeAddress.AddressProvince.ProvinceCode.ReferenceDataID,
```

Comment:
Consider renaming this field to `provinceTerritoryStateId` for the following reasons:
- Developers may be confused and think the value could be a [2 letter province/territory code](https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/completing-slips-summaries/financial-slips-summaries/return-investment-income-t5/provincial-territorial-codes.html) but it is actually a UUID
- Consistency with the regions lookup object: https://github.com/DTS-STN/canadian-dental-care-plan/blob/c3bf1024250f1e71383b4b01159a2ba95bbfcc9d/frontend/app/services/lookup-service.server.ts#L584

(same comment for `provinceCode` within `mailingAddress`)

Reply:
Makes sense. Updated: 3489b449c8a8d57369f2afe860d035d0deaf0e1e